### DISCUSSION TITLE
What is the purpose of metadata-base-ok.json?

### DISCUSSION THREAD

**Discussion by dalyles** [The-OpenROAD-Project/OpenROAD-flow-scripts#1844] [Q&A]
I see <code>metadata-base-ok.json</code> comes with prefilled values when i installed ORFS.
What is the purpose of this file and how does it play a role when running ORFS with <code>make</code>?
Does a new version of this file get generated after running <code>make</code>?

Thanks

**Comment by maliberty** [comment]
It represents a snapshot of the metrics used to generate the current rules-base.json.  It is not generated during each run and represents a snapshot in time.  `make metadata` will generate a metadata-base.json in reports from your run.

**Reply by dalyles** [reply]
Great, thanks for clarifying.

Does the file "do anything" besides contain a representation of metrics?
Like does it get fed into any part of the ORFS flow or is it just an output file for users to analyze?
Thanks

**Reply by maliberty** [reply]
It is just there for reference.  If you get a failure it is helpful to see the previous good values.
